apps/heft/src/plugins/SassTypingsPlugin/postcss-modules.d.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+// Add missing Definitely Typed typings
+declare module 'postcss-modules';
```

Comment:
@iclanton It seemed particularly difficult to import a package lacking a DefinitelyTyped package. Am I doing something wrong with importing/using postcss-modules (as far as typings goes)?

Reply:
We turned on a strict compiler check option that's probably causing this, but yeah, this is expected.